src/app/functions.js

```diff
@@ -0,0 +1,201 @@
+import i18n from '../i18n'
+
+const sumAll = numbers => {
+  let points = 0
+  numbers.forEach(number => {
+    points = points + number
+  })
+  return points
+}
+
+export const isThreeOfKind = numbers => {
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5) return false
+  if (isFullHouse(numbers)) return true
+  let sameNumbers = 0
+  const foundSameNumbers = []
+  sortedArray.reduce((a, b) => {
+    if (a === b) {
+      foundSameNumbers.push(b)
+      sameNumbers = sameNumbers + 1
+    }
+    return b
+  })
+  return sameNumbers === 2 && foundSameNumbers[0] === foundSameNumbers[1]
+}
+
+const threeOfKindPoints = numbers => {
+  return isThreeOfKind(numbers) ? sumAll(numbers) : 0
+}
+
+export const isFourOfKind = numbers => {
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5 || isFullHouse(numbers)) return false
+  let sameNumbers = 0
+  sortedArray.reduce((a, b) => {
+    if (a === b) sameNumbers = sameNumbers + 1
+    return b
+  })
+  return sameNumbers === 3
+}
+
+const fourOfKindPoints = numbers => {
+  return isFourOfKind(numbers) ? sumAll(numbers) : 0
+}
+
+export const isFullHouse = numbers => {
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5) return false
+  const isFirstTwoAreSame = sortedArray[0] === sortedArray[1]
+  const isLastTwoAreSame = sortedArray[3] === sortedArray[4]
+  const isFirstThreeAreSame = isFirstTwoAreSame && sortedArray[1] === sortedArray[2]
+  const isLastThreeAreSame = isLastTwoAreSame && sortedArray[2] === sortedArray[3]
+  if (isFirstThreeAreSame && isLastTwoAreSame && sortedArray[2] !== sortedArray[3]) {
+    return true
+  } else if (isFirstTwoAreSame && isLastThreeAreSame && sortedArray[1] !== sortedArray[2]) {
+    return true
+  } else {
+    return false
+  }
+}
+
+const fullHousePoints = numbers => {
+  return isFullHouse(numbers) ? 25 : 0
+}
+
+export const isSmallStraight = numbers => {
+  // 1-2-3-4
+  // 2-3-4-5
+  // 3-4-5-6
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5) return false
+  let consecutiveNumbers = 0
+  sortedArray.reduce((a, b) => {
+    if (a + 1 === b) consecutiveNumbers = consecutiveNumbers + 1
+    return b
+  })
```

Comment:
extracting this to a function like `getConsecutiveNumberCount = (numbers) => ...` would be an option for cleaner code and less redundancy